docs/content/2.deploy/0.index.md

```diff
@@ -27,19 +27,25 @@ When deploying to the production using CI/CD, Nitro tries to automatically detec
 - [vercel](/deploy/providers/vercel)
 - [zeabur](/deploy/providers/zeabur)
 
+::alert{type="info"}
+**Note:** Even with these presets, if you need to build your application locally (for development or testing purposes) targeting the provider environment, you still need to tell Nitro what preset it needs to use (you can see below how).
```

Comment:
Preview is not applied for all presets so for now at least it cannot be a general advice